components/camera/replaypcd/replaypcd.go

```diff
@@ -127,6 +137,12 @@ func (replay *pcdCamera) NextPointCloud(ctx context.Context) (pointcloud.PointCl
 		return nil, errors.New("session closed")
 	}
 
+	// Retrieve next cached data and remove from cache
+	if len(replay.cachedData) != 0 {
+		return replay.getDataFromCache()
+	}
+
```

Comment:
Could we add a comment like "If there's nothing left in the cache, download another batch"